doc/7/controllers/security/m-get-users/index.md

```diff
@@ -0,0 +1,41 @@
+---
+code: true
+type: page
+title: mGetUsers
+description: Gets multiple security users
+---
+
+# mGetUsers
+
+Gets multiple security users.
+
+<br />
+
+```js
+mGetUsers(kuids, [options]);
+```
+The route exists in `GET` and `POST`.
+By default, the SDK hits the `GET` one.
+You can force it to be `POST` in the `options`.
```

Comment:
This shouldn't be part of the route description, but should instead be a dedicated sub-chapter for the `verb` option below.

Something such as:

```
#### verb

When instantiated with a HTTP protocol object, the SDK uses the GET API by default for this API route.
You can set the `verb` option to `POST` to force the SDK to use the POST API instead.
```